### PR TITLE
🎨Maintenance: make redis client use the client name to ease debugging

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
- adds the client name when connecting to Redis DB
- in redis-commander (or redis-cli) one can hit `client list` in the console and see information about the connected clients
- cleanup of some tests that were not properly shutting down or not properly waiting for async tasks to finish... removed some warnings

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/director/requirements/_tools.txt
Comment (GitHK) [inline services/director/requirements/_tools.txt:1]: Q: why was this removed?

**Review by GitHK — APPROVED** [review]
very nice thanks

File: services/director/requirements/_tools.txt
Comment (sanderegg) [reply services/director/requirements/_tools.txt:1]: because this is wrong, as long as the dv-0 is not in python 3.11

Comment (sanderegg) [reply services/director/requirements/_tools.txt:1]: and it's only 3rd time I need to remove it

**Review by giancarloromeo — APPROVED** [review]

File: services/storage/src/simcore_service_storage/redis.py [hunk 1]
```diff
@@ -20,7 +21,9 @@ async def _setup(app: web.Application):
         assert settings.STORAGE_REDIS  # nosec
         redis_settings: RedisSettings = settings.STORAGE_REDIS
         redis_locks_dsn = redis_settings.build_redis_dsn(RedisDatabase.LOCKS)
-        app[_APP_REDIS_KEY] = client = RedisClientSDK(redis_locks_dsn)
+        app[_APP_REDIS_KEY] = client = RedisClientSDK(
+            redis_locks_dsn, client_name=APP_NAME
```
Comment (pcrespov) [inline services/storage/src/simcore_service_storage/redis.py:25]: MINOR: dont you want to distinguish between replicas as well? we might want to capture that in the app-name perhaps and then have it for free everywhere

**Review by pcrespov — APPROVED** [review]
thx

**Review by matusdrobuliak66 — APPROVED** [review]

File: services/storage/src/simcore_service_storage/redis.py (hunk 1 above)
Comment (sanderegg) [reply services/storage/src/simcore_service_storage/redis.py:25]: not sure about this fact. maybe sometimes yes, and sometimes no
